mattermost-plugin/webapp/src/components/rhsChannelBoardItem.test.tsx

```diff
@@ -42,7 +44,7 @@ describe('components/rhsChannelBoardItem', () => {
     })
 
     it('render board with menu open', async () => {
-        const board = createBoard()
+        const board = TestBlockFactory.createBoard()
```

Comment:
Guessing this was the reason for the snapshot changes above.

What was the reason behind changing to TestBlockFactory? If it was necessary for proper testing, then I would also change the other place we currently call `createBoard()`.
If not necessary, I would leave, as it causes unnecessary changes to snapshots which confuse the actually changes to this PR.

Reply:
That's great. Let's go ahead and update the other test to `TestBlockFactory.createBoard()` as well and eventually we can deprecate it...also may be a good community campaign to change it in other tests.